files/kerberos_ktadd_key_helper_test.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+d=$(mktemp -d)
+k="$d"/t.keytab
+p1=a/b@C.D
+p2=e/f@C.D
+
+hwclock --systohc
+
+ktadd() {
+	date="$1"
+	kvno="$2"
+	date -s "$1"
+
+	cat << EOF | ktutil
+addent -password -p $p1 -k $kvno -e arcfour-hmac
+a
+addent -password -p $p2 -k $kvno -e arcfour-hmac
```

Comment:
At a minimum, the script should have some comments at the beginning that documents the purpose of the script and that it's mainly used for testing and not needed for production. Please move this into a 'test-scripts' (or some better named) folder instead of 'files'. Given the 'test' in the file name, I thought that the script ran some check that was used by the puppet code. rspec tests would be ideal, but we would need to bootstrap the testing environment, possibly with something like test-kitchen or beaker. Adding support for test kitchen or beaker should be done as a separate pull request.